bigip/resource_bigip_ltm_snat.go

```diff
@@ -67,6 +67,13 @@ func resourceBigipLtmSnat() *schema.Resource {
 				Optional:    true,
 				Description: "Disables the SNAT on all VLANs.",
 			},
+			"vlans": {
+				Type:        schema.TypeSet,
```

Comment:
Any reason to use `TypeSet` here? why not simply `TypeList`?

Reply:
Ok, not sure it's a `TypeSet` there either.

In any case, if you keep it as `TypeSet`, since it's an aggregate type you need to error check when you set it in state.

here --> https://github.com/terraform-providers/terraform-provider-bigip/pull/40/files#diff-28144cac3e6bb762ff78904ab5c6569cR219